src/styles/blocks/star.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+@for $starCount from 1 through 5 {
+  .stars--#{starCount} .stars__star:nth-child(-n+#{$starCount}) {
```

Comment:
It seems like it doesn't work:

![image](https://user-images.githubusercontent.com/86923089/215337375-406f8ceb-6c53-4bf3-adda-01fd92111242.png)

That's also the reason tests fail
Please, fix it